src/modules/commentCounter.test.js

```diff
@@ -0,0 +1,20 @@
+const { countComments } = require('./commentCounter.js');
+
+describe('Count comments of a meal item', () => {
+  test('No comments', () => {
+    const arr = [];
+    expect(countComments(arr)).toBe(0);
+  });
+  test('Expect to get 2 comments back', () => {
+    const comments = [{
+      creation_date: '2022-10-26',
+      username: 'kenny',
```

Comment:
- [ ] When you create a new comment, it is expected that the page gets updated with the new comment. Your current implementation requires that the user clicks on the "Refresh Comments" button. Please implement this change just like the previous reviewer suggested🙏

Reply:
Thanks, I have done it now